src/main/java/Duke.java

```diff
@@ -1,10 +1,113 @@
+import java.util.Scanner;
+
 public class Duke {
+    static int noOfTasks = 0;
+    static String sectionDivider = "____________________________________________________________";
     public static void main(String[] args) {
+        Task[] TaskArray = new Task[100];
+        Task newItem = null;
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println(sectionDivider);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("Please type tasks to do OR (list) to list all the tasks OR (bye) to exit.");
+        Scanner sc= new Scanner(System.in);
+        while (true) {
+            String str = sc.nextLine();
```

Comment:
can consider using a general function to deal with each possible command (OOP way), instead of a sequential code.